### PR TITLE
ddl: Add a ddl_op file

### PR DESCRIPTION
**Pull request by zimulala**
Move DDL operations to ddl_op.go

### CODE REVIEW

File: ddl/ddl_op.go [hunk 1]
```diff
@@ -0,0 +1,1154 @@
...
+		BinlogInfo: &model.HistoryInfo{},
+		Args:       []interface{}{colName},
+	}
+
+	err = d.doDDLJob(ctx, job)
+	err = d.callHookOnChanged(err)
+	return errors.Trace(err)
+}
+
+// Modifiable checks if the 'origin' type can be modified to 'to' type with out the need to
```
Comment (hanfei1991) [inline ddl/ddl_op.go]: Modeifiable-> modifiable

**Comment by zimulala** [comment]
PTAL @hanfei1991 @tiancaiamao @coocood

**Comment by hanfei1991** [comment]
LGTM

**Comment by coocood** [comment]
I think change the new file name to `ddl_api.go` is better, those functions are called DDL statement executor.

**Pull request merged**